### ISSUE TITLE
twistedchecker cleanup in conch

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8200]
|[<img alt="mithrandi's avatar" src="https://avatars.githubusercontent.com/u/856430?s=50" width="50" height="50">](https://github.com/mithrandi)| @mithrandi reported|
|-|-|
|Trac ID|trac#8200|
|Type|defect|
|Created|2016-02-06 23:35:24Z|
|Branch|https://github.com/twisted/twisted/tree/conch-twistedchecker-cleanup-8200|

[#7413](https://github.com/twisted/twisted/issues/7413) managed to introduce some new twistedchecker failures, oops :(

<details><summary>Searchable metadata</summary>

```
trac-id__8200 8200
type__defect defect
reporter__mithrandi mithrandi
priority__normal normal
milestone__None None
branch__branches_conch_twistedchecker_cleanup_8200 branches/conch-twistedchecker-cleanup-8200
branch_author__mithrandi mithrandi
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__None None
time__1454801724330853 1454801724330853
changetime__1454881668423739 1454881668423739
version__None None
owner__mithrandi mithrandi
cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="mithrandi's avatar" src="https://avatars.githubusercontent.com/u/856430?s=50" width="50" height="50">](https://github.com/mithrandi)<a name="note_2"></a>|@mithrandi commented|
|-|-|

(In [46747]) Branching to conch-twistedchecker-cleanup-8200.

**Comment by twisted-trac** [comment]
|[<img alt="mithrandi's avatar" src="https://avatars.githubusercontent.com/u/856430?s=50" width="50" height="50">](https://github.com/mithrandi)|@mithrandi removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @mithrandi|
|-|-|


Required feedback:

1. There's a new pyflakes error, indicating a dead code path, in `intToBytes`.  Please fix.  (I realize that this is a copy/pasted function out of Cryptography, but given that it's only temporary, maybe just delete the code path so it doesn't harm our coverage numbers).

Things to consider for the future:

1. You might want to install https://github.com/glyph/python-docstring-mode in the editor of your choice.  (Is that vim?  I really need to do a release for vim.)  At the very least this will result in smaller diffs when I edit the same source file in the future :).

Please address the one required point and merge.  Glad to see such a significant a reduction in twistedchecker errors :=).

**Comment by twisted-trac** [comment]
|[<img alt="mithrandi's avatar" src="https://avatars.githubusercontent.com/u/856430?s=50" width="50" height="50">](https://github.com/mithrandi)<a name="note_5"></a>|@mithrandi set status to `closed`|
|-|-|

(In [46753]) Merge conch-twistedchecker-cleanup-8200: Formatting fixes and cleanups.

Author: idnar
Reviewer: glyph
Fixes: [#8200](https://github.com/twisted/twisted/issues/8200)

Also fix a compatibility issue with Cryptography 1.0.
